bundles/aws/build.gradle.kts

```diff
@@ -63,14 +61,12 @@ tasks.withType(ShadowJar::class.java) {
     exclude(dependency("org.slf4j:slf4j-api"))
   }
 
-  relocate("com.amazonaws", "org.apache.gravitino.aws.shaded.com.amazonaws")
   relocate("com.fasterxml.jackson", "org.apache.gravitino.aws.shaded.com.fasterxml.jackson")
   relocate("com.google.common", "org.apache.gravitino.aws.shaded.com.google.common")
   relocate("com.google.errorprone", "org.apache.gravitino.aws.shaded.com.google.errorprone")
   relocate("com.google.thirdparty", "org.apache.gravitino.aws.shaded.com.google.thirdparty")
   relocate("io.netty", "org.apache.gravitino.aws.shaded.io.netty")
   relocate("org.apache.commons", "org.apache.gravitino.aws.shaded.org.apache.commons")
-  relocate("org.apache.hadoop.fs.s3a", "org.apache.gravitino.aws.shaded.org.apache.hadoop.fs.s3a")
```

Comment:
This change also fixes issue #8041. Please add the issue number to the PR title and description.